rom/dev/src/flow/update_reset.rs

```diff
@@ -81,9 +81,14 @@ impl UpdateResetFlow {
         Self::copy_regions();
         report_boot_status(UpdateResetOverwriteManifestComplete.into());
 
+        // Return the old FHT from DCCM
+        let fht = FirmwareHandoffTable::try_load()
```

Comment:
Can the load and entry point addresses of RT be different than the previous in the new FW bundle? If yes, the values of these in FHT would be different from the actual load address of the new RT in ICCM.

Reply:
Hmm. The ICCM address could change, but I don't think the DataVault handle can change. So we probably don't need to update the FHT. I'll update this.

However, we should be overwriting the load address in DataVault. But when I try to call `write_warm_reset_entry4(WarmResetEntry4::RtLoadAddr, manifest.runtime.load_addr)` during update reset, I get a Load/Store fault. I would expect these datavault entries to be unlocked. Maybe a simulator issue?

Reply:
Changed to just not change FHT.

For the second part of the comment, opened https://github.com/chipsalliance/caliptra-sw/issues/518